frontend/test/metabase/containers/SaveQuestionModal.unit.spec.js

```diff
@@ -111,6 +108,39 @@ function fillForm({ name, description }) {
   }
 }
 
+function checkCall(
+  mockFn,
+  { question, name, id, description, collection_id } = {},
+) {
+  const call = mockFn.mock.calls[0];
+  expect(call.length).toBe(1);
+  expect(call[0]).toBeInstanceOf(Question);
+
+  let newQuestion = question;
+
+  if (typeof id !== "undefined") {
+    newQuestion = newQuestion.setId(id);
+  }
+  if (typeof name !== "undefined") {
+    newQuestion = newQuestion.setDisplayName(name);
+  }
+  if (typeof description !== "undefined") {
+    newQuestion = newQuestion.setDescription(description);
+  }
+  if (typeof collection_id !== "undefined") {
+    newQuestion = newQuestion.setCollectionId(collection_id);
+  }
+
+  // Check a few specific properties first for clearer debugging.
+  expect(call[0].id()).toBe(newQuestion.id());
+  expect(call[0].displayName()).toBe(newQuestion.displayName());
+  expect(call[0].description()).toBe(newQuestion.description());
+  expect(call[0].collectionId()).toEqual(newQuestion.collectionId());
+
+  // Then checking the overall isEqual() for completeness.
+  expect(call[0].isEqual(newQuestion)).toBe(true);
```

Comment:
nit: maybe assign `call[0]` to a variable to make it clear it's a question?

```js
const [question] = call
expect(question.id()).toBe(newQuestion.id())
```

Reply:
Done.